protos/monocle/protob/metric.proto

```diff
@@ -35,20 +35,30 @@ message GetRequest {
   }
 }
 
-message Histo {
+message HistoInt {
   string date = 1;
   uint32 count = 2;
 }
 
-message HistoStat {
-  repeated Histo histo = 1;
+message HistoFloat {
+  string date = 1;
+  float count = 2;
+}
+
+message HistoIntStat {
+  repeated HistoInt histo = 1;
+}
+
+message HistoFloatStat {
+  repeated HistoFloat histo = 1;
 }
 
 message GetResponse {
   oneof result {
     string error = 1;
     float float_value = 2;
     int32 int_value = 3;
-    HistoStat histo_value = 4;
+    HistoIntStat histo_int_value = 4;
```

Comment:
I guess we could remove an indirection by using:
```suggestion
    repeated HistoInt histo_int = 4;
```

Reply:
In a "oneof" clause "repeated" keyword if not authorized unfortunately.